cockpit/src/main/java/com/polidea/cockpit/paramsedition/ParamsEditionModel.kt

```diff
@@ -0,0 +1,33 @@
+package com.polidea.cockpit.paramsedition
+
+import com.polidea.cockpit.core.CockpitParam
+import com.polidea.cockpit.manager.CockpitManager
+import com.polidea.cockpit.utils.getParam
+
+class ParamsEditionModel {
+
+    private var paramsCopy: List<CockpitParam<Any>> = CockpitManager.getParamsCopy()
+
+    val size : Int
+        get() = paramsCopy.size
+
+    fun <T : Any> getParamAt(position: Int): CockpitParam<T> =
+            paramsCopy.getParam(position)
+
+    fun <T : Any> setValue(position: Int, newValue: T) {
+        paramsCopy[position].value = newValue
+    }
+
+    fun restoreValue(position: Int) {
+        val param = paramsCopy[position]
```

Comment:
How about making it into one line?

Reply:
We use `param` in two places in 
`param.value = CockpitManager.getParamDefaultValue(param.name)`
To me it's more readable. Let's discuss it with Wojtek.